### ISSUE TITLE
AirTerminal:SingleDuct:ConstantVolume:NoReheat not respecting Maximum Air Flow Rate input

### ISSUE THREAD

**Issue by eringold** [NREL/EnergyPlus#7744]
Issue overview
--------------
`SimConstVolNoReheat` appears to have no checks for user input Maximum Air Flow Rate:

https://github.com/NREL/EnergyPlus/blob/588fe91ef15aedef7c68a25117c1e27ec8293a43/src/EnergyPlus/SingleDuct.cc#L4721-L4783

![image](https://user-images.githubusercontent.com/10799524/74049901-58144e00-4992-11ea-98ff-b3beb2ee5145.png)

![image](https://user-images.githubusercontent.com/10799524/74049911-5e0a2f00-4992-11ea-9170-a86df1c16140.png)

I can email the defect file if needed. 

### Details
Some additional details for this issue (if relevant):
 - Windows x64
 - EnergyPlus v9.2

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by mjwitte** [comment]
@eringold Those checks are supposed to be happening here in [`SingleDuct::InitSys`](https://github.com/NREL/EnergyPlus/blob/588fe91ef15aedef7c68a25117c1e27ec8293a43/src/EnergyPlus/SingleDuct.cc#L2256-L2288). Please provide the idf file.

**Comment by eringold** [comment]
@mjwitte I emailed you the files, and thanks for correcting me on the relevant code section. 

For the record, I did some additional investigation: In the initial defect file (7744_initial.idf), a DOAS system serves a number off zones directly through AirTerminal:SingleDuct:ConstantVolume:NoReheats, as well as one zone with an AirTerminal:SingleDuct:Mixer supplying to a fan coil unit. In that file, I see the terminal air flow rates greater than their hard-sized maximums.

In the second file (7744_revised.idf), I removed the zone and the AirTerminal:SingleDuct:Mixer from the loop. This results in the ConstantVolume:NoReheats respecting the max flow rate inputs. 

The DOAS unit is auto-sized; the only thing hard-sized is the terminal max flow. It seems that, perhaps because of https://github.com/NREL/EnergyPlus/issues/6242, the DOAS in the initial file is over-delivering to the other zones because the FCU zone flow is throttled when the FCU is not operating. But I’m not sure why the DOAS doesn’t reduce its flow to meet the terminal maximums.
